### ISSUE TITLE
TFFA team colours bug

### ISSUE THREAD

**Issue by BlindOraclex** [taysta/TaystJK#83]
TFFA team colours are bugged so that the colours are not forced. Sometimes it is just default FFA colours, sometimes it is even red player models with blue sabers, on blue team. This just depends on what the person originally chose for their player model.

A short-term fix is to use cg_forcemodel 1 and you can even use cg_forceenemymodel and team colours will apply (the client's player model will remain bugged however). This again breaks upon changing map, vid_restarting etc. but can be turned off and on again whenever to temporarily fix it.

Usually occurs after changing map, vid_restarting or sometimes upon first connection to a TFFA server. Also can be replicated on devmap.

**Comment by Alereon** [comment]
Could you try to be a little bit more specific?

Does it have to do with saber colors? skin colors?
Any specific cvars that affect this? (not the one's you mention as a fix)

Do you have the exact steps to reproduce this?

**Comment by BlindOraclex** [comment]
Sure.

Saber colours are fine and not affected by the bug (blue team all keep blue sabers for example).

Skin colours are what I mean by the colours not being forced. At some point, the client's model will have default colours usually and other players will have a variety (probably depending on what they have picked in the menu). I would guess if they were on the blue team and picked their skin originally, when they move to the red team, they keep their blue skin which can be pretty confusing.

In addition to this, not sure if it's related but on base jamp little symbols appear above the players depending on their team. These don't appear at all but can't tell if this is due to bug or not since the bug is so common to happen.

Specific cvars to affect? Not that I know of, I tested it with a fully default cfg for example too and it still occurs. Might be worth mentioning that I've been hearing this bug happen to everyone I know who has used this eternal fork, so it shouldn't be too hard to reproduce in that aspect.

If it helps at all I think the first time I noticed this happening was maybe around 3-5 releases ago. I remember in earlier releases it was not a problem.

In terms of steps, I can't really say exact steps because it's so common that it happens. Many times when I first join a tffa game and I am in spec this bug happens immediately. 

If I wanted to force the bug to happen, I could only suggest:
-/Devmap mp/ffa3
-/g_gametype 6
-add bots so there are players on each team
-/vid_restart

I'd be surprised if the bug doesn't happen after these steps.
